projects/clr/hipamd/include/hip/amd_detail/amd_device_functions.h

```diff
@@ -726,6 +726,14 @@ __device__ inline __attribute__((convergent)) int __syncthreads_or(int predicate
   return __ockl_wgred_or_i32(!!predicate);
 }
 
+#if (defined(__gfx942__) || defined(__gfx950__))
```

Comment:
Something for another day, but is it possible to start moving away from these in general, except where they are absolutely necessary?